$NetBSD: patch-configure.ac,v 1.1 2018/05/04 15:33:22 leot Exp $

Also check for OPENSSL_init_ssl (fixes build with OpenSSL 1.1.0).

From upstream commit 070cee8d1efba1ca260777a204fba7cdfe676ca5.

--- configure.ac.orig	2015-09-14 14:40:44.000000000 +0000
+++ configure.ac
@@ -83,11 +83,16 @@ if test "x$found_libcrypto" = xno; then
 	AC_MSG_ERROR("libcrypto not found")
 fi
 AC_SEARCH_LIBS(
-	SSL_library_init,
+	OPENSSL_init_ssl,
 	[ssl],
 	found_libssl=yes,
 	found_libssl=no
 )
+AC_SEARCH_LIBS(
+	SSL_library_init,
+	[ssl],
+	found_libssl=yes,
+)
 if test "x$found_libssl" = xno; then
 	AC_MSG_ERROR("libssl not found")
 fi
